renovate.json

```diff
@@ -400,6 +400,32 @@
       "enabled": true,
       "minimumReleaseAge": "7 days"
     },
+    {
+      "groupName": "@elastic/kibana-management dependencies",
+      "matchDepNames": [
+        "@elastic/request-converter",
+        "@mapbox/vector-tile",
+        "cronstrue",
+        "file-saver",
+        "moment-duration-format",
+        "@mapbox/vector-tile",
+        "@types/mapbox__vector-tile",
+        "@types/moment-duration-format"
```

Comment:
These ones are also used bit by the monitoring plugin, should we separate it and add both teams as reviewers?

Reply:
I'll move this one to its own group with shared ownership.

Reply:
https://github.com/elastic/kibana/pull/202776/commits/e6a6353ae970a07164213bb008458c1e482d0c2c